_posts/2022-12-12-trino-summit-2022-starburst-recap.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Federating them all on Starburst Galaxy"
+author: "Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/starburst.jpg
+---
+
+I have been reading all the wonderful posts by our awesome
```

Comment:
I don't know how strongly I feel about it, but my thought is that it _might_ be good to introduce yourself in this first paragraph if the post is first-person perspective.

Reply:
I was trying to make the first paragraph my subtle intro without being like "hi im monica"